source/tutorials/nix-language.md

```diff
@@ -380,14 +380,32 @@ Example:
 let
   a = 1;
 in
-a + a
+  a + a
 ```
 
 ```{code-block}
 :class: value
 2
 ```
 
+Attempting to use the value outside of the binding produces an error:
+
+```{code-block} nix
+:class: expression
+a
+```
+
+```{code-block}
+:class: value
+error: undefined variable 'a'
+
+       at «string»:1:1:
+
+            1| a
+             | ^
+```
+
+
```

Comment:
I feel that this section is a bit redundant, as it's only showing that we can't refer to a variable that hasn't been defined.

Reply:
I agree. Do you think it would be appropriate to add commentary along the lines of "`let` is similar to curly brace scoping in other languages" with information on the similarities and differences? Or is that too much for this section of the tutorial?